### PR TITLE
Add setUpTestData for models

### PR DESCRIPTION
**Pull request by maximeconnolly**
Add setup for test in member models

### CODE REVIEW

**Comment by maximeconnolly** [comment]
@gzsierra @cmccauley can someone check this pr. This is only a little test script to make sure the models are saving correctly.

**Comment by gzsierra** [comment]
Can you clean your code please ?
Like remove all no necessary changing lines

File: django/metaclub/member/migrations/0002_remove_member_user.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10 on 2016-08-21 17:44
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
```
Comment (gzsierra) [inline django/metaclub/member/migrations/0002_remove_member_user.py]: remove changing line

File: django/metaclub/member/models.py [hunk 2]
```diff
@@ -17,8 +17,6 @@ class Meta:
         max_length=50
     )
 
-    user = models.ForeignKey(User)
-
```
Comment (gzsierra) [inline django/metaclub/member/models.py]: remove changing line

File: django/metaclub/member/migrations/0002_remove_member_user.py (hunk 1 above)
Comment (maximest-pierre) [inline django/metaclub/member/migrations/0002_remove_member_user.py]: can automated generation of file

**Pull request merged**